src/view/range.js

```diff
@@ -82,6 +87,60 @@ export default class Range {
 	}
 
 	/**
+	 * Creates a new range with the same content, but containing all surrendering tags with no semantic meaning.
```

Comment:
BTW. I understand that by "semantic" you meant "not being a data". Data elements are `view.Text` and `view.EmptyElement` right now, right? Others are aggregating elements (`ContainerElement`) styling elements (`AttributeElement`) and UI elements (`UIElement`).